.eslintrc.json

```diff
@@ -51,6 +51,7 @@
     "generator-star-spacing": "off",
     "import/no-extraneous-dependencies": "off",
     "import/newline-after-import": "off",
+    "class-methods-use-this": "off",
```

Comment:
Where did you run into problems with this rule?

Reply:
Here's an example: https://github.com/cumulus-nasa/cumulus/blob/master/packages/ingest/granule.js#L290

cmrjs package has a few examples: https://github.com/cumulus-nasa/cumulus/blob/master/packages/cmrjs/index.js

ingest/protocol has examples: https://github.com/cumulus-nasa/cumulus/blob/master/packages/ingest/protocol.js

I started out having eslint ignore specific functions like those ^, but found enough instances of it that I gave up.

I think the assertion that rule is making is that those methods should be functions that are not attached to classes. I agree with that assertion, but this code base has enough class methods that don't use `this` that it doesn't seem like a good use of effort to refactor them all, mostly because the examples I'm finding aren't particularly useful separated from the classes they're attached to.